Kitodo-API/src/main/java/org/kitodo/api/dataformat/MediaUnit.java

```diff
@@ -176,6 +204,13 @@ public boolean equals(Object obj) {
         } else if (!orderlabel.equals(other.orderlabel)) {
```

Comment:
This comment is about code not changed here but I find it important:

`if (obj == null) {
            return false;
        }
        if (!(obj instanceof MediaUnit)) {
            return false;
        }`

Do you know that _!(obj instanceof MediaUnit)_ checks also against null? You could skip first check.

Reply:
Above all, your comment is not on the affected line. I changed it.